### PR TITLE
Start Material docs

### PR DESCRIPTION
**Pull request by jhanstra**
Beginning to put together some documentation on how we'll use Material. This is a place to put any Material components we use in MyUW, quick links to their documentation, and anything worth noting about how we use them or how we develop with them.

### CODE REVIEW

File: docs/markdown/material.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<md-button class="md-raised">Button</md-button>
+<md-button class="md-raised md-primary">Primary</md-button>
+<md-button ng-disabled="true" class="md-raised md-primary">Disabled</md-button>
+<md-button class="md-raised md-warn">Warn</md-button>
+```
++ **[Fab Buttons](https://material.angularjs.org/latest/demo/button)**: Used for links inside of widgets. 
+```
+<md-button class="md-fab md-mini md-primary" aria-label="Use Android">
+  <md-icon md-svg-src="img/icons/android.svg" style="color: greenyellow;"></md-icon>
+</md-button>
```
Comment (timlevett) [inline docs/markdown/material.md]: Looking at the material docs, you are actually not supposed to use the FAB buttons in this way. Checkout https://www.google.com/design/spec/components/buttons-floating-action-button.html for details. Not sure what we will do about the circle buttons in widgets, maybe something like fab, but makes one rethink the circle button approach.

Comment (timlevett) [inline docs/markdown/material.md]: Maybe we could suggest doing something like list with avatar where the avatar is the fa icon? https://www.google.com/design/spec/components/lists.html#lists-specs

File: docs/markdown/home.md [hunk 2]
```diff
@@ -16,6 +16,7 @@
 + [Widgets](#/md/widgets) : All things widgets
 
 ### Styling
++ [Material](#/md/material) : Information on how we use Material Design
```
Comment (timlevett) [inline docs/markdown/home.md]: Tweak to say "going to use" since this isn't true for the current release.

File: docs/markdown/material.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+### Material Design
+
```
Comment (timlevett) [inline docs/markdown/material.md]: Maybe add a "timeline" section. Says what the rollout plan is

3.x -> material + ui-bootstrap w/ bs3
4.x -> material only

File: docs/markdown/home.md (hunk 2 above)
Comment (jhanstra) [inline docs/markdown/home.md]: Good advice, updated

File: docs/markdown/material.md (hunk 3 above)
Comment (jhanstra) [inline docs/markdown/material.md]: I added a section, feel free to edit

**Pull request merged**